zebrad/src/components/mempool/crawler/tests/prop.rs

```diff
@@ -7,17 +7,18 @@ use proptest::{
     prelude::*,
 };
 use tokio::time;
+use tower::BoxError;
```

Comment:
Suggestion (3/4): Can we use `zebrad::BoxError` instead?

Reply:
Done.